src/Symfony/Bundle/FrameworkBundle/EventListener/NoRoutingConfigurationSubscriber.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpKernel\Kernel;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\Routing\Exception\NoConfigurationException;
+
+/**
+ * Show welcome message if no routing configuration is available.
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class NoRoutingConfigurationSubscriber implements EventSubscriberInterface
```

Comment:
What about adding this logic to the existing `RouterListener` instead of adding a new one?

Reply:
Done.